_posts/randy/2022-07-03-all-old-movies-are-garbage.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title:  "All Old Movies Are Garbage (In the End)"
```

Comment:
Add another line below this with 

```
author: randy
```

This allows us to filter by all of your articles

Reply:
poor example of one